Travel Project v1/Travel Project v1/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Event;
+using InterfaceImplementation;
+using Interfaces;
+using SomeStuff;
+using System;
+
+namespace Travel_Project_v1
+{
+    class Program
+    {
+
+        //Dictionary
+        static void Main(string[] args)
+        {
+
+            //Event
+
+            double d = double.Parse(Console.ReadLine());
+            var priceFlight = new EventLogic();
+            EventHandling showPrice = new EventHandling();
```

Comment:
it is not event handling, as event is never raised and noone subscribes to the event